### PR TITLE
Makes mentions actually the number of mentions

### PR DESCRIPTION
**Pull request by Zachary-Dernar**
Previously it was the number of days that an article showed up

### CODE REVIEW

File: views/MentionsAndSentiments/index.jsx [hunk 1]
```diff
@@ -89,6 +89,16 @@ export default React.createClass({
     this.setState({ mentions });
   },
 
+  calculateMentionCount(data) {
+    var totalSum = 0;
```
Comment (maniax89) [inline views/MentionsAndSentiments/index.jsx]: could use `let`

File: views/MentionsAndSentiments/index.jsx [hunk 2]
```diff
@@ -89,6 +89,16 @@ export default React.createClass({
...
+    for (let article of data) {
```
Comment (maniax89) [inline views/MentionsAndSentiments/index.jsx]: could use `data.forEach((article) => { ...`

**Review by maniax89 — COMMENTED** [review]
add/update a test for this in the `MentionsAndSentiments.test.js` file

File: test/views/MentionsAndSentiments/test.index.js [hunk 3]
```diff
@@ -120,11 +120,19 @@ describe('<MentionsAndSentiments />', () => {
...
-  it('Orders results by number of "mentions"', () => {
+  it('Orders results by number of mentions', () => {
     let wrapper = shallow(<MentionsAndSentiments query={query_sample} mentions={mentions_sample} />);
     let firstMention = wrapper.find(Accordion).nodes[0].props.header.props.children[0].props.children;
     let secondMention = wrapper.find(Accordion).nodes[1].props.header.props.children[0].props.children;
     assert.equal(firstMention, 'Sample Company + Company Two');
     assert.equal(secondMention, 'Sample Company + Company One');
   });
+
+  it('Reports number of mentions, not number of days a mention occured', () => {
```
Comment (maniax89) [inline test/views/MentionsAndSentiments/test.index.js]: no reason to mention the previous behavior - that concept should disappear. instead use a test name that explains the expected behavior - maybe something like "calculates a summation of positive, negative, and neutral mentions"

File: views/MentionsAndSentiments/index.jsx [hunk 4]
```diff
@@ -90,12 +90,12 @@ export default React.createClass({
...
   calculateMentionCount(data) {
-    var totalSum = 0;
-    for (let article of data) {
+    let totalSum = 0;
+    data.forEach((article) => {
       totalSum += article.positive;
       totalSum += article.neutral;
       totalSum += article.negative;
-    }
+    });
```
Comment (kognate) [inline views/MentionsAndSentiments/index.jsx]: You could express this via a reduce statement more cleanly.

```
let sums = data.reduce((acc, item) => {
   return [acc[0] + item.positive, acc[1] + item.neutral, acc[2] + item.negative]
}, [0,0,0])
```

**Review by maniax89 — APPROVED** [review]

**Pull request merged**